spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -71,6 +71,14 @@ def get_parser():
         default=30,
         help="Degree of smoothing for centerline fitting. Only for -centerline-algo {bspline, linear}."
     )
+    optional.add_argument(
+        "-centerline-soft",
+        metavar=Metavar.int,
+        type=int,
+        choices=[0, 1],
+        default=0,
+        help="Binary or soft centerline. 0 = binarized, 1 = soft"
+    )
```

Comment:
Since the script is called `sct_get_centerline`, I don't think we necessarily need to include the word "centerline" in the argument name?

It should be relatively understandable if the argument is just called `-soft`, I think?

Reply:
Yeah. Good point! This is exactly what we were considering. But since other arguments such as [-centerline-algo](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/7cb30ddbaa88ded9c3ae6250c72832f5c40f4d64/spinalcordtoolbox/scripts/sct_get_centerline.py#L62) and [-centerline-smooth](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/7cb30ddbaa88ded9c3ae6250c72832f5c40f4d64/spinalcordtoolbox/scripts/sct_get_centerline.py#L68) also contains the word "centerline", we decided to include the word also to the `-centerline-soft` argument.

Reply:
Ahh, very good point!